posts/MichelaRocchettiBlogPost/Michela_JSoC.qmd

```diff
@@ -0,0 +1,142 @@
+It includes several projects that provide a wide variety of datasets.
+The information and data collected by *IPUMS* are useful for comparative research, as well as for the analysis of individuals in their life contexts.
+These data can be used to create a more comprehensive dataset that will facilitate research on the social determinants of health for different types of diseases, social communities and geographical areas. 
+
+![](./IPUMS_grid_logo.png)
+
+> To learn more about IPUMS, visit the [website](https://www.ipums.org) 
+
+The primary objectives of this proposal is to:
+
+1. Develop a native Julia package to interact with the APIs available around the datasets *IPUMS* provides.
+
+2. Provide useful utilities within this package for manipulating *IPUMS* datasets.
+
+3. Compose this package with the wider Julia ecosystem to enable novel research in health, economics and more. 
+
+# Tasks and Goals
+
+1. Expand some of the functionality developed in `ipumsr`  *IPUMS* NHGIS
+    - Create a link between OpenAPI documentation and the functions internally used in IPUMS.jl:
+    updating already present function, determine if updating is needed and test them
+    - Develop functionality similar to the get_metadata_nghis function present in ipumsr
+
+2. Update *IPUMS* documentation
+    - Set-up and deploy of DocumenterVitepress.jl  
+    - Write a blog post on how IPUMS.jl can compose within the ecosystem.
+
+
+# Workflow 
+
+The first task was to migrate documents from Documenter to DocumenterVitepress.The aim of this issue is to support the significant refactoring underway across JuliaHealth, 
+aimed at improving the discoverability and cohesion of the JuliaHealth ecosystem, particularly in relation to documentation.This issue is intended to provide guidance on migrating the
+the IPUMS.jl documentation site from Documenter.jl to DocumenterVitepress.jl, in order to create a more attractive entry point for new Julia users interested in health research within the Julia community.
+To accomplish this task, a dependency of DocumenterVitepress was added to the docs directory of the IPUMS.jl repository. 
+Once this was done, the Documenter.jl make.jl file was migrated into a DocumenterVitepress.jl make.jl file. Working on the make.jl file, pages, paragraphs and sub-paragraphs were added in order to
+structure for the web page explaining the IPUMS.jl package. With this in mind, several pages have been added:
+  1. Home: to explain the main purpose of this work and to give general information.
+  2. Workflows: to explain the working process behind it.
+  3. How to: to give general information on how to do things.
+  4. Tutorials: some tutorials about what you can do with IPUMS.jl and how to make it work. 
+  5. Examples: some examples of activities.
+  6. Mission: to better explain why the IPUMS.jl package is useful for the community, and its final goal and possible impact.
+  7. References: references used to write the pages.
```

Comment:
Could you cut this down some?

Reply:
Do you mean the bullet points or the text part?

Reply:
Both please!